src/models/Follower.model.ts

```diff
@@ -0,0 +1,41 @@
+import { ObjectId } from "mongodb";
+import mongoose, { Schema, Document } from "mongoose";
+import paginate from "mongoose-paginate-v2";
+
+export interface Follower extends Document {
+  followerId: ObjectId;
+  ownerId: ObjectId;
+  createdAt: string;
+  updatedAt?: string;
+}
+
+const FollowerSchema: Schema = new Schema<Follower>({
+  followerId: {
+    type: mongoose.Schema.Types.ObjectId,
```

Comment:
```suggestion
    type: Schema.Types.ObjectId,
```
https://mongoosejs.com/docs/typescript.html